packages/scripts/environments/production.yml

```diff
@@ -65,6 +65,22 @@ cronJobs:
         memory: 5Gi
     backoffLimit: 3
 
+  - name: materialize-scrubbed-message-stats
+    schedule: "0 */2 * * *" # Every 2 hours
+    command: ["npm", "run", "scripts:materializeScrubbedMessagesStats:latest"]
+    env:
+      MONGODB_DATABASE_NAME: docs-chatbot-staging
+    envSecrets:
+      MONGODB_CONNECTION_URI: docs-chatbot-staging
```

Comment:
why is there a production script that points to our staging DB?

Reply:
Because I originally copied this in the staging env file and forgot to update 🤦🏻 Will fix!